logisland-core/logisland-framework/logisland-utils/src/main/java/com/hurence/logisland/util/runner/TestRunner.java

```diff
@@ -46,7 +46,7 @@
     /**
      * Performs the operation.
      */
-    void run();
+    void run() throws InitializationException;
```

Comment:
You should remove this as we decided to throw exception as RunTimeException instead